mediapicker/source-camera/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+    id 'org.jetbrains.kotlin.plugin.parcelize'
+    id 'org.jetbrains.kotlin.kapt'
+    id 'dagger.hilt.android.plugin'
+    id 'maven-publish'
+    id "com.automattic.android.publish-to-s3"
+}
+
+android {
+    namespace "org.wordpress.android.mediapicker.source.camera"
+
+    compileSdkVersion rootProject.compileSdkVersion
+
+    defaultConfig {
+        minSdkVersion rootProject.minSdkVersion
+        targetSdkVersion rootProject.targetSdkVersion
+    }
+
+    lint {
+        warningsAsErrors true
+    }
+}
+
+dependencies {
+    implementation mediaPickerDomainDependency
+
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$kotlinxCoroutinesVersion"
+
+    implementation "com.google.dagger:hilt-android:$gradle.ext.daggerVersion"
+    implementation "com.google.dagger:hilt-android-compiler:$gradle.ext.daggerVersion"
+    kapt "com.google.dagger:hilt-compiler:$gradle.ext.daggerVersion"
+}
+
+project.afterEvaluate {
+    publishing {
+        publications {
+            SourceDevicePublication(MavenPublication) {
```

Comment:
Warning (⚠️): I think it is better to use a different name for this publication, not sure if that is a warning comment or not, but didn't want to risk it and then have our publishing failing for some reason, or worse being somehow inconsistent just because we are using the same name twice: `SourceCameraPublication` 🤷

Reply:
Yeaaa, just me trying to be on the safe side... 😊